### PR TITLE
Resource/stake/requeset

### PR DESCRIPTION
**Pull request by basakkang**
1. 로그아웃시 페이지 reload 되도록 변경
2. delegatebw action 보내기 추가

### CODE REVIEW

File: app/frontend/src/elm/Component/Main/Page/Resource/Stake.elm [hunk 1]
```diff
@@ -124,17 +138,19 @@ type Message
...
     | StakePercentage PercentageOfLiquid
     | OpenStakeAmountModal
+    | SubmitAction
     | ModalMessage StakeAmountMessage
 
 
 type StakeAmountMessage
     = CpuAmountInput String
     | NetAmountInput String
+    | OkButton
```
Comment (Heejaejoo) [inline app/frontend/src/elm/Component/Main/Page/Resource/Stake.elm]: Click OK?
Let's define message starting with verbs.

File: app/frontend/src/elm/Component/Main/Page/Resource/Stake.elm [hunk 2]
```diff
@@ -184,6 +200,13 @@ update message ({ delegatebw, totalQuantity, distributionRatio, stakeAmountModal
         OpenStakeAmountModal ->
             ( { model | isStakeAmountModalOpened = True }, Cmd.none )
 
+        SubmitAction ->
+            let
+                cmd =
+                    { delegatebw | from = accountName, receiver = accountName } |> Action.Delegatebw |> encodeAction |> Port.pushAction
```
Comment (Heejaejoo) [inline app/frontend/src/elm/Component/Main/Page/Resource/Stake.elm]: You may want to clear model after submitting action.

File: app/frontend/src/elm/Component/Main/Page/Resource/Stake.elm (hunk 1 above)
Comment (Heejaejoo) [inline app/frontend/src/elm/Component/Main/Page/Resource/Stake.elm]: Click OK?
Let's define messages starting with verbs.

File: test/frontend/elm/Test/Component/Main/Page/Resource/Stake.elm [hunk 3]
```diff
@@ -21,7 +21,12 @@ import Util.Validation as Validation
 tests : Test
 tests =
     describe "Page.SearchKey module"
-        [ describe "getPercentageOfLiquid"
+        [ describe "initModel"
+            [ test "delegatebw.transfer always should be 0 when from == receiver" <|
```
Comment (Heejaejoo) [inline test/frontend/elm/Test/Component/Main/Page/Resource/Stake.elm]: should always be?

File: app/frontend/src/elm/Component/Main/Page/Resource/Stake.elm (hunk 1 above)
Comment (basakkang) [reply app/frontend/src/elm/Component/Main/Page/Resource/Stake.elm]: 🆗

File: app/frontend/src/elm/Component/Main/Page/Resource/Stake.elm (hunk 2 above)
Comment (basakkang) [reply app/frontend/src/elm/Component/Main/Page/Resource/Stake.elm]: OK I will do it at next PR :)

File: test/frontend/elm/Test/Component/Main/Page/Resource/Stake.elm (hunk 3 above)
Comment (basakkang) [reply test/frontend/elm/Test/Component/Main/Page/Resource/Stake.elm]: 👍

**Pull request merged**